### ISSUE TITLE
Demo link not working

### ISSUE THREAD

**Issue by namelessperson0** [Angristan/node-chat-app#7]
The demo link https://angristan-node-chat-app.herokuapp.com is not live

**Comment by angristan** [comment]
Yes, I disabled it because it was using too much dyno time on Heroku

**Comment by namelessperson0** [comment]
Cool. Removed link in https://github.com/angristan/node-chat-app/pull/9
